Pod/Classes/Format/CardNumberFormatter.swift

```diff
@@ -164,26 +162,26 @@ public final class CardNumberFormatter {
      - parameter textField: The text field whose text should be changed.
      - parameter string:    The new string. This might be unformatted or badly formatted and will be formatted properly before being inserted into `textField`.
      */
-    public func replaceRangeFormatted(range: NSRange, inTextField textField: UITextField, withString string: String) {
-        let newValueUnformatted = self.unformattedCardNumber(NSString(string: textField.text ?? "").stringByReplacingCharactersInRange(range, withString: string))
+    public func replaceRangeFormatted(_ range: NSRange, inTextField textField: UITextField, withString string: String) {
```

Comment:
Might be possible to make this more readable with something like `format(range: NSRange, in textField: UITextField, andReplaceWith string: String)`